modules/statics/src/coins.ts

```diff
@@ -1253,6 +1260,26 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.ETH,
     OAS_FEATURES
   ),
+  account(
+    'bac24d8c-0f8f-4530-a63c-bc52458acf95',
+    'coredao',
+    'coredaochain',
```

Comment:
Is this name ok like this? Maybe `CoreDAO chain` or whatever is the official name

Reply:
this name is picked after discussion, so we are planning keep the consistency for EVM coins